account_fiscal_position_vies_warning/models/account_move.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2025 Moduon Team S.L. <info@moduon.team>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, models
+from odoo.exceptions import ValidationError
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def action_post(self):
+        if self.env.company.vat_check_vies:  # Verify VAT Numbers is True
```

Comment:
- Early returns tend to be more readable
- Use comments when they complement information :)
```suggestion
        if not self.env.company.vat_check_vies: 
            return super().action_post()
```